### DISCUSSION TITLE
Lightbox : How to display only the first image of the gallery in the document and all others images only in the gallery.

### DISCUSSION THREAD

**Discussion by wilcar** [quarto-dev/quarto-cli#8783] [Q&A]
I really like the gallery mode of the lightbox. 
 have several images in a lightbox gallery. Is it possible to only display the first image in my document to launch the gallery which will display the rest of the images? 
In other words, how do I hide the gallery images in the document except for the first one ?

**Comment by mcanouil** [comment]
I don't think the library can do that.
See <https://biati-digital.github.io/glightbox/>.

Note that you can embed any JavaScript library (as a Quarto extension or using the `include-` YAML keys).

It should be easy to find some HTML example online that you can adapt.
